quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -138,14 +140,15 @@ function verifyZitiVersionExists {
 
   unset ZITI_BINARIES_VERSION
 
-  ziticurl="$(curl -s https://api.github.com/repos/openziti/ziti/releases/tags/"${ZITI_VERSION_OVERRIDE}")"
+  ziticurl="$(curl -s https://${GITHUB_TOKEN:+${GITHUB_TOKEN}@}api.github.com/repos/openziti/ziti/releases/tags/"${ZITI_VERSION_OVERRIDE}")"
```

Comment:
Expands to empty string if not defined, otherwise expands as `${GITHUB_TOKEN}@`

Reply:
I added this because I keep running into the GitHub rate limit which breaks `expressInstall`.

Reply:
> I added this because I keep running into the GitHub rate limit which breaks expressInstall.

this is the sort of comment that fits best in the code imo

Reply:
I'll revert this change unless someone can test it with macOS bash v3.

Reply:
did you want to keep the change or revert it? my point is that IF you keep it. you should add your comment INTO the script:

> Expands to empty string if not defined, otherwise expands as ${GITHUB_TOKEN}@

Reply:
what i mean is, if you ever feel the need to comment on code like this - i think that's a pretty clear indicator to take a moment and consider if that comment belongs "in" the code. in this case - i feel like it definitely would be useful to have in the script